ambry-store/src/test/java/com/github/ambry/store/CompactionPolicyTest.java

```diff
@@ -82,6 +94,135 @@ public CompactionPolicyTest(String compactionPolicyFactoryStr) throws Exception
     mockBlobStoreStats = blobStore.getBlobStoreStats();
     CompactionPolicyFactory compactionPolicyFactory = Utils.getObj(compactionPolicyFactoryStr, config, time);
     compactionPolicy = compactionPolicyFactory.getCompactionPolicy();
+    dataDirPath = Paths.get(MOUNT_PATH).toAbsolutePath();
+  }
+
+  private void cleanupBackupFiles() throws IOException {
```

Comment:
Minor: please move private utility methods to bottom.

Reply:
Updated.